conf/ds-image-environment.yaml.sample

```diff
@@ -13,7 +13,9 @@
 config:
 
   # Production machine setup
-  backend:
-    db: 'http://realmachine:8090/foobar/'
-    user: 'real_dbuser'
-    password: 'real_dbpassword'
+  imageservers:
+    # Full request URL sample: http://example.com:1234/iipsrv/iipsrv.fcgi?FIF=/my_images/hello.tif&CVT=jpg
+    iip: 'http://example.com:1234/iipsrv/iipsrv.fcgi'
+
```

Comment:
Please add sample (i.e. non-working URL) for Deepzoom

Reply:
Samples has been added